x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/insights/endpoint_insights.tsx

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I'd prefer if we renamed these files to `workflow_insights` so it's in line with the feature name.